### PR TITLE
Fix id_desc link in layer.md

### PR DESCRIPTION
**Pull request by coolljt0725**
Signed-off-by: Lei Jitang <leijitang@huawei.com>

### CODE REVIEW

**Comment by philips** [comment]
id_desc doesn't exist either.

File: layer.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 An example of creating an Image Filesystem Changeset follows.
 
 An image root filesystem is first created as an empty directory.
-Here is the initial empty directory structure for the a changeset using the randomly-generated directory name `c3167915dc9d` ([actual layer DiffIDs are generated based on the content](#id_desc)).
+Here is the initial empty directory structure for the a changeset using the randomly-generated directory name `c3167915dc9d` ([actual layer DiffIDs are generated based on the content](#serialization.md#terminology)).
```
Comment (wking) [inline layer.md]: `#serialization.md#terminology` → `serialization.md#terminology`

**Comment by wking** [comment]
On Thu, Sep 01, 2016 at 05:56:19AM -0700, Brandon Philips wrote:
> id_desc doesn't exist either.

The serialization file's “Terminology” section does define “Layer
DiffID”, so I think linking to that section is fine.

**Comment by coolljt0725** [comment]
@philips  `serialization.md#terminology` has description of `layer DiffIDs`, so I think it's ok to link to there

**Comment by jonboulle** [comment]
This is addressed in #255 which is being actively worked on now so I think we should just defer to there.

**Pull request closed without merging**